db/README.txt

```diff
@@ -1,2 +1,6 @@
 This directory contains schema and query files used by various tools
 to interact with the databases.
+
+`schema_tmp_metadata` defines the schema for the temporary metadata table used as a staging area.
+This table includes measurement metadata, which can be obtained using irisctl, while other metadata are sourced from the configuration file and added during the final insertion.
+Using this staging table helps improve efficiency by avoiding multiple calls to BigQuery, streamlining the overall process.
```

Comment:
In normal text files, it's best to keep the lines under 72 characters.  You can use `fmt` to format your lines.  For example, here's a formatted version with slight edits for better readability.

`schema_tmp_metadata` defines the structure of the temporary metadata
table, which serves as a staging area for converting schema formats.
This table stores measurement metadata, retrievable via `irisctl`,
while additional metadata is sourced from the configuration file
and incorporated during the final insertion.  Using this staging
table enhances efficiency by minimizing the need for multiple
BigQuery calls, thereby streamlining the entire process.